connectors/src/connectors/google_drive/index.ts

```diff
@@ -92,6 +93,85 @@ export async function createGoogleDriveConnector(
   }
 }
 
+export async function updateGoogleDriveConnector(
+  connectorId: ModelId,
+  {
+    connectionId,
+  }: {
+    connectionId?: NangoConnectionId | null;
+  }
+): Promise<Result<string, ConnectorsAPIErrorResponse>> {
+  if (!NANGO_GOOGLE_DRIVE_CONNECTOR_ID) {
+    throw new Error("NANGO_GOOGLE_DRIVE_CONNECTOR_ID not set");
+  }
+
+  const c = await Connector.findOne({
+    where: {
+      id: connectorId,
+    },
+  });
+  if (!c) {
+    logger.error({ connectorId }, "Connector not found");
+    return new Err({
+      error: {
+        message: "Connector not found",
+        type: "connector_not_found",
+      },
+    } as ConnectorsAPIErrorResponse);
```

Comment:
Is the `as` needed here ? If so, why ?

Reply:
Thank you for noticing, It's not, I removed it from all other connectors and forgot for this one 🤦🏻‍♀️